### PR TITLE
Doc: Issue68 add set search_path to env_setup.md

### LINKED ISSUE #68: Doc: Modify setup instruction to set search_path on ENV_SETUP_INSTRUCTION.md
## Description
As a developer,
I need to make sure the set search_path is included inside ENV_SETUP_INSTRUCTION.md,
so that I can make sure other contributors can set up their local environment successfully.

## Mocks
NA

### Update [Required]
- [ ] Added relevant changes to [BIT Development Environment Setup Instruction Wiki ](https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup) on setting search path to bitschema into ENV_SETUP_INSTRUCTION.md

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description

Add set search_path to ENV_SETUP_INSTRUCTION.md to reflect changes on WIKI page [BIT Development Environment Setup Instructions](https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup)

Fixes #68 

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by mtreacy002** [comment]
Update @anitab-org/bridgeintech-maintainers . I've updated the WIKI BIT Development Environment Setup Instruction to include setting search_path to bitschema as suggested on our mentors meeting last night. This PR is to make sure the ENV_SETUP_INSTRUCTION.md on .github folder is reflecting the changes. Can you please review and approve this? Thanks

File: .github/ENV_SETUP_INSTRUCTION.md [hunk 1]
```diff
@@ -51,15 +50,42 @@ To do this, click on `bit_schema` on the left sidebar, go to `Properties` (top n
...
+1. Run the command below to make sure we have 2 schemas already in the database
+> $ psql -c '\dn;' -U postgres -d bit_schema
+
+you should see the following
+<img width="971" alt="Screen Shot 2020-06-28 at 2 56 28 pm" src="https://user-images.githubusercontent.com/29667122/85938323-92858400-b94f-11ea-803b-cf2cea70d94f.png">
+
+1. Run the next command to show the existing search_path
+> $ psql -c 'show search_path;' -U postgres -d bit_schema_test
+
+// TO DO: ADD AN IMAGE SHOWING THE INITIAL SEARCH PATH. IT SHOULD ONLY HAVE the `public` schema
```
Comment (meenakshi-dhanani) [inline .github/ENV_SETUP_INSTRUCTION.md]: Why is there a TODO comment here?

**Comment by meenakshi-dhanani** [comment]
I'm not sure why TODO comments are committed, if you have something to say to another contributor who should add images, please mention the line number to them in the PR, I would not prefer comments like this

**Comment by mtreacy002** [comment]
Ok, I'll remove them both in Wiki link and the doc then'll  push the code change. 👍

**Comment by mtreacy002** [comment]
Update @meenakshi-dhanani and @anitab-org/bridgeintech-maintainers . I have removed the TO DO comments and replaced with line numbers on relevant issues description.

**Review by meenakshi-dhanani — APPROVED** [review]

**Review by foongminwong — APPROVED** [review]
reviewed and looks good to me

**Pull request merged**